localizer_psychopy/localizer_exp

```diff
@@ -0,0 +1,81 @@
+# import psychopy modules
+from psychopy import visual, core, event, sound, gui, data
+
+#set parent directory
+parent_dir = "/Users/Kayleigh/fmriproject/localizer_psychopy/"
```

Comment:
It might be better to use relative path here so that the script can be easily run across computers without changing the directory every time. I'd do `parent_dir = "./"`